flepimop/gempyor_pkg/tests/statistics/test_statistic_class.py

```diff
@@ -248,9 +248,7 @@ def test_unsupported_regularizations_value_error(
             ]
             if reg_name not in ["forecast", "allsubpop"]
         )
-        with pytest.raises(
-            ValueError, match=rf"^Unsupported regularization\: {unsupported_name}$"
-        ):
+        with pytest.raises(ValueError, match=r"^Unsupported regularization \[received:.*$"):
```

Comment:
Again, prefer this matches the offending value offered rather than the precise message.